### PR TITLE
PR templates

### PR DESCRIPTION
**Pull request by 4ydan**
In case we want to build upon the idea of having different PR templates. As for now I didnt spent much time thinking about it and it might look redundant, but if done right, we can gain something from this approach for sure.
Issue: #533

## Basics

- [x] I added a line to [/doc/CHANGELOG.md](/doc/CHANGELOG.md)
- [x] The PR is rebased with current master.
- [x] Details of what you changed are in commit messages.
- [x] References to issues, e.g. `close #X`, are in the commit messages.
- [x] The buildserver is happy.

## Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added unit tests for my code
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
      (exceptions are documented)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] Examples are well chosen and understandable

### CODE REVIEW

File: doc/CHANGELOG.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Syntax: `- short text describing the change _(Your Name)_`
 - _()_
 - _()_
 - _()_
-- _()_
+- DEV: Add PR templates.
```
Comment (markus2330) [inline doc/CHANGELOG.md:25]: Here you conflict with yourself (#537)

File: .github/PULL_REQUESTS/enhancement_template.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<!--
```
Comment (markus2330) [inline .github/PULL_REQUESTS/enhancement_template.md:1]: See .github/CONTRIBUTING.md for which types of PRs we have (Branch names). This should probably renamed from enhancement -> feature.

Probably CI-related should get their own PR template, too?

**Review by markus2330 — CHANGES_REQUESTED** [review]
Yeah, we can try PR templates too. But they should basically resemble the branch name conventions we have (per branch name one PR template).

File: .github/PULL_REQUESTS/enhancement_template.md (hunk 2 above)
Comment (4ydan) [reply .github/PULL_REQUESTS/enhancement_template.md:1]: Good hint, thanks.

File: .github/PULL_REQUESTS/bug_fix_template.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+name: Bug fix
+about: For PermaplanT bug fixes
+title: ''
+labels: 'bug'
+assignees: ''
+---
+
```
Comment (markus2330) [inline .github/PULL_REQUESTS/bug_fix_template.md:8]: At least "Basics" should be also here.

File: .github/PULL_REQUESTS/bug_fix_template.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+**Description**
+
+<!-- A description of the bug and the fix. Also link the issue if it exists. -->
```
Comment (markus2330) [inline .github/PULL_REQUESTS/bug_fix_template.md:11]: The link should be in the ChangeLog. (And the issue should exist, so "if it exists" can be removed.)

**Comment by 4ydan** [comment]
Thanks, what do we wanna add to the release and meeting template?

**Comment by 4ydan** [comment]
What do we think including the basics before opening a pull request, so it is not duplicated across all templates.

**Comment by 4ydan** [comment]
I am really not happy with this solution and I would maybe rather go back to how it was before. I underestimated the difficulty of this problem.

**Comment by markus2330** [comment]
> Thanks, what do we wanna add to the release and meeting template?

Not needed there.

> I am really not happy with this solution and I would maybe rather go back to how it was before. I underestimated the difficulty of this problem.

You mean not merging this at all?

> What do we think including the basics before opening a pull request, so it is not duplicated across all templates.

How would that work?

**Comment by 4ydan** [comment]
I don't feel like doing this anymore, I don't think anymore that a hierarchical solution is necessary. We can come back to this another day maybe, when it's really needed.

**Comment by markus2330** [comment]
I agree, let us drop this.

**Pull request closed without merging**